### PR TITLE
CNNGNS-61 - Stand Alone Galleries in GNS

### PR DESCRIPTION
**Pull request by zoitec**
Feed-generator: Move blacklist to environment variable.

Fixes: CNNGNS-61 - Parse stand alone galleries.

PR_URL:

Review-By: Sean Joseph sean.joseph@turner
Review-By: James Young james.young@turner

### CODE REVIEW

File: lib/feed-generator.js [hunk 1]
```diff
@@ -632,32 +632,99 @@ class FeedGenerator {
...
+                                    feed += `<media:credit role="author" scheme="urn:ebu"><![CDATA[${slide.credit}]]></media:credit>`;
+                                }
+
+                                feed += '</media:content>';
+
+                                // start handling body content
+                                feed += '<content:encoded><![CDATA[';
+
+                                // analytics tracking
+                                feed += `<amp-pixel src="${config.get('adbpTrackingURL')}/1/G.4--NS/RANDOM?pageName=cnn%3Ac%3ACANONICAL_PATH&events=event26&v26=cnn%3Ac%3ACANONICAL_PATH&server=CANONICAL_HOST&v29=CANONICAL_HOST&ch=CANONICAL_URL&v27=CANONICAL_URL&c28=CANONICAL_URL&v28=CANONICAL_URL&c14=google%20newsstand&v14=google%20newsstand&c26=CANONICAL_HOST%2FCANONICAL_URL&c30=cnn%20domestic&v30=cnn%20domestic&c32=adbp%3Acontent&v32=adbp%3Acontent&c33=article%20read&v33=article%20read&c35=googlenewsstand.1%3A20160930&v35=googlenewsstand.1%3A20160930&c46=PAGE_VIEW_ID&v46=PAGE_VIEW_ID&c64=cnn%20news&v64=cnn%20news&g=AMPDOC_URL&vid=CLIENT_ID&r=DOCUMENT_REFERRER"></amp-pixel>`;
```
Comment (jamsyoung) [inline lib/feed-generator.js]: I know this amp-pixel has to be there.. wow, thats long though!

Comment (CRHain88) [inline lib/feed-generator.js]: Do you have a utility that converts an object to a query string?

File: lib/feed-generator.js [hunk 2]
```diff
@@ -632,32 +632,99 @@ class FeedGenerator {
...
-                //             feed += `<media:description type="plain"><![CDATA[${description}]]></media:description>`;
-                //         }
-                //
-                //         if (slide.credit) {
-                //             feed += `<media:credit role="author" scheme="urn:ebu"><![CDATA[${slide.credit}]]></media:credit>`;
-                //         }
-                //
-                //         feed += '</media:content>';
-                //     });
-                // }
```
Comment (zoitec) [inline lib/feed-generator.js]: Done

**Pull request merged**